handlers/handlers.go

```diff
@@ -24,7 +25,7 @@ type ClientError interface {
 
 // RenderClient is an interface with methods for require for rendering a template
 type RenderClient interface {
-	Do(string, []byte) ([]byte, error)
+	Page(w io.Writer, page interface{}, templateName string)
```

Comment:
The interface method has changed as we now directly pass the page model data and the name of the template we want to map it to, to the renderer library